app/assets/stylesheets/src/career_site/_main.scss

```diff
@@ -15,7 +15,8 @@ body {
   min-height: calc(100% - #{$footer-min-height});
 }
 
-#back_on_all_positions {
+a,
+a:visited:not(.btn, .icon-button-component, .nav-link, .dropdown-item, .badge) {
```

Comment:
Let's don't make these changes since they will be in conflict with tabler branch and rewriting visited link color.

Please, add style:

> .grecaptcha-badge { 
>     visibility: hidden !important;
> }